src/olympia/users/admin.py

```diff
@@ -169,7 +173,24 @@ def reset_api_key_view(self, request, object_id, extra_context=None):
         if not acl.action_allowed(request, amo.permissions.USERS_EDIT):
             return HttpResponseForbidden()
 
-        self.reset_api_key_action(request, [obj])
+        self.reset_api_key_action(
+            request, UserProfile.objects.filter(pk=obj.pk))
+
+        return HttpResponseRedirect('../')
+
+    def reset_session_view(self, request, object_id, extra_context=None):
+        if request.method != 'POST':
+            return HttpResponseNotAllowed(['POST'])
+
+        obj = self.get_object(request, unquote(object_id))
+        if obj is None:
+            raise Http404()
+
+        if not acl.action_allowed(request, amo.permissions.USERS_EDIT):
+            return HttpResponseForbidden()
+
+        self.reset_session_action(
+            request, UserProfile.objects.filter(pk=obj.pk))
 
         return HttpResponseRedirect('../')
```

Comment:
Same here although this was clearly there before.

Reply:
https://github.com/mozilla/addons-server/pull/13763/commits/20814c702404863bbb56f2e2b73d8d609b1791f9